ckanext/nextgeoss/public/communityfeedback.js

```diff
@@ -152,16 +168,30 @@ function communityFeedback(catalogueID, catalogueNamespace, title)
     };
 
     if (abstract.length == 0) {
-      abstract = "No abstract available"
+      try {
```

Comment:
The changes in 9da7254 mean that if, for example, a user leaves the `abstract` field empty, the feedback will display `Abstract:` rather than `Abstract: No abstract available` in browsers like Opera or Chrome that require the non-namespaced version of the parsing.

The intended behavior, which still works in Firefox and some versions of Chrome, is that if a field like `abstract` is empty, then the feedback should display `Abstract: No abstract available` rather than just `Abstract:` with no clarification.

The new behavior in Opera and versions of Chrome that don't support the namespaced parsing is to display only `Abstract:` if the `abstract` has no value. `Abstract: No abstract available` is only displayed if parsing fails.

(The behavior is still correct in Firefox because the empty field means the length check fails, triggering the second attempt at parsing, which fails because it won't work with Firefox, causing the `No abstract available` message to be displayed.)

The `comment` field doesn't seem to be affected, but I'm not sure why.

I suggest structuring the try/catch sequences like this: 
 
```javascript
try {
  var abstract = feedback.getElementsByTagName("guf\:abstract")[0]
    .getElementsByTagName("gco\:CharacterString")[0].textContent;
} catch(err) {
  try {
    abstract = feedback.getElementsByTagName("abstract")[0]
      .getElementsByTagName("CharacterString")[0].textContent;
  } catch(err) {
    abstract = "";
  };
 };
 if (abstract.length == 0) {
  abstract = "No abstract available"
 };
```
Then the message will be displayed whenever the field is empty _or_ missing from the XML.

Reply:
We'll handle the separately later.